packages/odata-common/src/entity-base.ts

```diff
@@ -74,10 +74,10 @@ export abstract class EntityBase {
 
   abstract readonly _oDataVersion: any;
 
-  constructor(private schema: Record<string, any>) {
-    nonEnumerable(this, 'schema');
+  constructor(readonly _entityApi: any) {
```

Comment:
[q] I know we show in all the tutorials to use the `entityBuilder` to create entities. However, if someone used the constructor for some obscure reason directly this would break now. I would add a compatibility note. Also I would add the @internal annotation on the constructor.

Reply:
You're right, I was also wondering about backwards compatibility, my understanding was that typically users don't call the constructor. I'll add an annotation and a note.